### PR TITLE
Developement

### PR DESCRIPTION
**Pull request by lilleonz**

### CODE REVIEW

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @lilleonz @codingAngarita,
Good job. Here is the checklist for your project.
- To consider the project 'complete' it must satisfy the following points:
## General
- [x] The project has a descriptive README.md
## Navigation bar
* [x] It should use the `<nav>` HTML tag
* [x] It should have seven separate containers (`<div>` is recommended)
## Main Content
* [x] All content should be wrapped with `<main>` HTML tags
* [x] Every children of main should be a `<section>` HTML tag pair
* [x] Every children of section should be an `<article>` HTML tag pair
## Footer 
* [x] It should have an unorganized list `<ul>`
* [x] The links in the footer should be list items `<li>` positioned using the `float` or `flex` display property

Fix the unchecked list and please send another request to re-review this project.

Thanks 🙏

File: index.html [hunk 1]
```diff
@@ -5,8 +5,183 @@
...
+                        <li><a href="#">Spicy</a></li>
+                        <li><a href="#">Salami</a></li>
+                        <li><a href="#">Spicy</a></li>
+                        <li><a href="#">Salami</a></li>
+                        <li><a href="#">Spicy</a></li>
+                        <li><a href="#">Salami</a></li>
+                    </ul>
+                </nav>
+                <footer class="footer">
+                    <p>Spicy jalapeno bacon ipsum dolor amet <br> swine ham ground round kevin tail salami <br> tenderloin cow burgdoggen. Short loin andouille fatback <br> venison pork belly. Filet mignon brisket alcatra pork belly,<br> boudin spare ribs pork loin fatback chicken pork <br>chop kevin prosciutto.</p>
```
Comment (Torres-ssf) [inline index.html]: As was said in the first review, you should use `<ul>` here and list all links with `<li>` items. Don't forget that they must be centralized inside the footer, both horizontally and vertically.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
Hi @lilleonz. You are almost there. As was said the `<footer>` must have `<ul>` and all links inside `<li>` items. Please kindly make this change and submit for code review again.

File: index.html (hunk 1 above)
Comment (lilleonz) [reply index.html]: footer has been updated with links

**Comment by maelfosso** [comment]
Hello,

# Nice Job

You can now submit a project completion form.

**Pull request merged**